homeassistant/package_constraints.txt

```diff
@@ -9,6 +9,7 @@ aiohttp==2.2.5
 async_timeout==1.4.0
 chardet==3.0.4
 astral==1.4
+defusedxml==0.5.0
```

Comment:
Don't do that as core requirements. Make it only for platform

Reply:
@pvizeli sorry about that, done.